### ISSUE TITLE
[FEATURE] Checkboxes

### ISSUE THREAD

**Issue by Swexti** [elkowar/eww#118]
## Description of the requested feature
Just a simple checkbox, they would be good.

## Proposed configuration syntax
`<checkbox checked="what happens when checked, if possible"/>`

# Additional context
no

**Comment by elkowar** [comment]
it would make sense to have a
```xml
<checkbox onchecked="notify-send 'checkbox checked'" onunchecked="notify-send 'checkbox unchecked'"/>
```
thing, imo. but yea, sounds good!

**Comment by elkowar** [comment]
Has been implemented in #119
